Simplified.xcodeproj/project.pbxproj

```diff
@@ -2384,6 +2388,7 @@
 				739E6120244A0D8600D00301 /* FirebaseAnalytics.framework in Frameworks */,
 				731B2B25244A2B6E00A7A649 /* R2Shared.framework in Frameworks */,
 				739E6121244A0D8600D00301 /* nanopb.framework in Frameworks */,
+				170E5FBE25AFDD65007D6DE6 /* ZIPFoundation.framework in Frameworks */,
```

Comment:
was this missing only on the R2dev target or also other targets? In the latter case, that's weird, because the build was passing anyway.

Reply:
Yeah, it builds fine but crash on launch, at least on the simulator.
It's missing on all the targets, I added it on the targets that use r2-lcp.